vizro-core/src/vizro/models/_components/_form.py

```diff
@@ -12,7 +12,7 @@
 from vizro.models import VizroBaseModel
 from vizro.models._components.form import Checklist, Dropdown, RadioItems, RangeSlider, Slider
 from vizro.models._layout import set_layout
-from vizro.models._models_utils import _log_call, _validate_min_length
+from vizro.models._models_utils import _log_call, check_captured_callable, validate_min_length
```

Comment:
What's the difference between a private function from a private module, and a non private function from a private model @antonymilne - ie why does `_log_call` have a `_`?

Reply:
No difference really. The private function from a private module is just more obvious that it's private because you don't need to look at the import to see that it's private. Probably I would make all of these private with the `_` in front of them for that reason, but it doesn't matter much.

Reply:
I would leave as is in this PR for now, because it was driving me a bit crazy that it was inconsistent in this file 😄 I would propose to do a separate PR where we consistently go through each file and implement above then.